egs2/aishell/asr1/conf/train_asr_streaming_conformer.yaml

```diff
@@ -0,0 +1,95 @@
+# network architecture
```

Comment:
Could you emphasize which part is important for the streaming configuration?
(e.g., `contextual_block_conformer`, right?)
Please add such comments.

Reply:
Sure!